Wireframe/index.html

```diff
@@ -1,13 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
   </head>
+
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+       <main>
+      <header class="header-container">
+        <h1>Welcome to Git</h1>
+        <p>
+          A distributed version control system, where every developer gets their
+          local repository with full commit history.
+        </p>
+      </header>
+
+      <section class="article1">
+        <img class="article1_img" src="/Wireframe/git2.png">
+        <h2>What is Git?</h2>
+        <p>
+          Git is a free and open source distributed version control system
+          designed to handle everything from small to very large projects with
+          speed and efficiency. Git is easy to learn and has a tiny footprint
+          with lightning fast performance. It outclasses SCM tools like
+          Subversion, CVS, Perforce, and ClearCase with features like cheap
+          local branching, convenient staging areas, and multiple workflows.
+        </p>
+        <a href="https://git-scm.com/" target="_blank">READ MORE</a>
+        
+      </section>
+
+      <div class="section-container">
+          <section class="article">
+            <img class="article2_img" src="/Wireframe/git1.png">
+          <h2>Why do Developers need Git?</h2>
+          <p>
+            Git is all about efficiency. For developers, it eliminates
+            everything from the time wasted passing commits over a network
+            connection to the man hours required to integrate changes in a
+            centralized version control system. It even makes better use of
+            junior developers by giving them a safe environment to work in.
+          </p>
+      
+            <a href="https://www.atlassian.com/git/tutorials/why-git"
+              target="_blank">
+              READ MORE</a>
+          
+        </section>
+        <section class="article">
+          <img class="article3_img" src="/Wireframe/git3.png">
```

Comment:
img element must have an alt attribute, except under certain conditions. 
```suggestion
<img class="article1_img" src="/Wireframe/git3.png" alt"adds image description here">
```

More about the img element here: https://www.w3.org/WAI/tutorials/images/